content/company-info-and-process/values/index.md

```diff
@@ -46,6 +46,24 @@ With agency comes responsibility; teammates are expected to take initiative and
 
 While [decisions should be made eagerly](../communication/decisions.md), this doesn't mean that you should make unilateral decisions when others should be involved. As an [open company](../about-sourcegraph/index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. It is not a bad outcome if your proposal is not accepted or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one! We're a startup, and if we take the safe path, we will most likely fail.
 
+#### Be iterative
+
+We try to ship small things fast, instead of big things slowly. Being iterative allows us to avoid unecesary long planning, avoid deadlocks, and derisk what we do faster, increasing the value we create for customers. Postponing shipping things leads to them being outdated, creates complexity in managing what's not been shipped, and delays learning.
+Discussions around large changes tend to be slow and deadlock, whereas small steps are much more likely to lead to a decision. It's always easier to re-evaluate after we have learnt by taking a first step.
+
+Having a strong vision and strategy allows us to be iterative: it gives us the framework that we need to move in the right direction, and allows us to plan less and iterate faster instead.
+
+In practice this means we:
+
+- cut scope to get things out faster
+- don't hide work in progress from customers as long as the change is better than the previous state
+- start a proposal, instead of waiting to organize a meeting to collect input
+- try to make reversible decisions, that allow us to change our mind if we need to
+
+Being iterative helps us have high agency, because small steps are easier to take and don't require approval. This is why it is a sub-principle of high agency. It also allows us to be more open and transparent, because we can learn in public by consyantly shipping small improvements.
```

Comment:
```suggestion
Being iterative helps us have high agency, because small steps are easier to take and don't require approval. This is why it is a sub-principle of high agency. It also allows us to be more open and transparent, because we can learn in public by constantly shipping small improvements.
```